apps/src/templates/manageStudents/PrintLoginCards.jsx

```diff
@@ -13,33 +13,36 @@ const styles = {
 
 export default class PrintLoginCards extends Component {
   static propTypes = {
-    sectionId: PropTypes.number
+    sectionId: PropTypes.number,
+    entryPointForMetrics: PropTypes.oneOf(
```

Comment:
would it be possible/acceptable to change 'entryPointForMetrics' to something else, like metricsCategory? When I see 'entry point' in a variable name, I assume the variable name is a callback because an entry point usually corresponds to the first instructions executed by a program that is being launched: https://en.wikipedia.org/wiki/Entry_point

I think I get what you're going for here - the metrics category for this component represents which 'entry point' from our application this component was used from - but I think its about equally clear to name the variable metricsCategory or similar, and have the reader infer what the metrics category means from context. I think this leaves a little bit less room for confusion

Reply:
I went with entrypoint since the [spec](https://docs.google.com/document/d/1YaC8yPL4QUNshj0zXmCZJ1o-FcGYb1Dexv9J0-acHuM/edit#bookmark=id.h8c5lrcxz0v) used entrypoint and that's the field name that we'll be querying for.  In this case it's clear it's not some other meaning of entry point since this line defines it as one of the metrics categories for the buttons.